tidb-cloud/size-your-cluster.md

```diff
@@ -37,6 +37,30 @@ The supported node sizes include the following:
 > - TiFlash is unavailable.
 
 ### TiDB node quantity
+The performance of the TiDB node is generally linearly scalable, but there is some performance degradation when there are a larger number of nodes. We estimate this performance degradation by increasing the performance degradation coefficient by 5% every time the number of nodes reaches 8. 
+For example, when there are 9 TiDB nodes, the overall performance is `9 * (1 - 5%) = 8.55` times the performance of a single TiDB node. And when there are 16 TiDB nodes, the overall performance is `16 * (1 - 10%) = 14.4` times the performance of a single TiDB node.
+
+Different workloads involve queries with varying read-write ratios and have different performance in the specified latency for a TiDB node.
+The performance of the 8 vCPU, 16 GiB TiDB node in different workloads:
+
+    | workload | TiDB performance (QPS) |
+    |----------|------------------------|
+    | Read     | 18,900                 |
+    | Mixed    | 15,500                 |
+    | Write    | 18,000                 |
+
+We estimate the number of TiDB nodes based on the workload type, overall expected performance (QPS), and the performance of a single TiDB node under different workloads:
+ `node num = ceil( overall expected perfromance / performance per node * (1 - performance degradation coefficient) )`
+
+In the absence of performance degradation, the performance of 16 vCPU, 32 GiB TiDB nodes is roughly twice that of 8 vCPU, 16 GiB TiDB nodes. If the number of nodes is too large, it is recommended to choose 16 vCPU, 32 GiB TiDB nodes as this will require fewer nodes and have a smaller performance degradation coefficient.
```

Comment:
```suggestion
When the number of TiDB nodes is less than 8, the performance deviation coefficient is nearly 0%, so the performance of 16 vCPU, 32 GiB TiDB nodes is roughly twice that of 8 vCPU, 16 GiB TiDB nodes. If the number of TiDB nodes exceeds 8, it is recommended to choose 16 vCPU, 32 GiB TiDB nodes as this will require fewer nodes, which means smaller performance deviation coefficient.
```